.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @KnpLabs/Mentoring
```

Comment:
it seems that there's an issue with the code owners

Reply:
yeah I couldn't found out why :man_shrugging: